manifests/capacityscheduling/crd.yaml

```diff
@@ -1,54 +1,88 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: elasticquotas.scheduling.sigs.k8s.io
   annotations:
-    "api-approved.kubernetes.io": "https://github.com/kubernetes-sigs/scheduler-plugins/pull/52"
+    api-approved.kubernetes.io: https://github.com/kubernetes-sigs/scheduler-plugins/pull/52 # edited manually
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: elasticquotas.scheduling.sigs.k8s.io
 spec:
   group: scheduling.sigs.k8s.io
   names:
+    kind: ElasticQuota
+    listKind: ElasticQuotaList
     plural: elasticquotas
     singular: elasticquota
     kind: ElasticQuota
```

Comment:
This is duplicated.

Reply:
Removed the duplicated entry.

Reply:
Well, the comment was meant to delete the latter `kind: ElasticQuota` so as to keep the fields sorted alphabetically; otherwise the codegren diffs. Also `listKind` should be restored.

(you can take a look at the CI failure log: click the "raw build-log.txt" link on https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_scheduler-plugins/222/pull-scheduler-plugins-unit-test-master/1435806643223793664)

Reply:
Oh my bad, I misinterpreted that. I've updated the code with latest push.